src/hometask8/Product.java

```diff
@@ -0,0 +1,61 @@
+package hometask8;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+public class Product {
+    private String type;
```

Comment:
It's better to use enum for type

Reply:
I did not get how can I use it here

Reply:
Create new enum eg.
public enum Type {
  BOOK,
  PEN,
  ..
}
And use instead of string